### ISSUE TITLE
Vizlab skeleton function in vizlab

### ISSUE THREAD

**Issue by jiwalker-usgs** [USGS-VIZLAB/vizlab#3]
create a skeleton function based off the package skeleton idea that will create all the starting bits for developing a new visualization

**Comment by jiwalker-usgs** [comment]
There is something in place for this, but I'm not quite sure it is complete.

**Comment by jiwalker-usgs** [comment]
I looked at this again, and it seems to be doing enough to call done.  We will want to look back at this in future visualizations to make sure the skeleton sets up a reasonable set of defaults.

**Comment by eread-usgs** [comment]
Verify this creates the necessary directories if it doesn't exist.

**Comment by lindsaycarr** [comment]
Need license to be created automatically

**Comment by lindsaycarr** [comment]
What vizSkeleton creates with an empty project:

```
data/
images/
figures/
layout/templates
layout/js
layout/css
scripts/visualize
scripts/read
scripts/process
scripts/fetch
viz.yaml
.gitignore
LICENSE
```

LICENSE was added in #62

**Comment by eread-usgs** [comment]
create .gitempty or .empty that allows you to check in an empty directory

**Comment by eread-usgs** [comment]
make a function to: profile.yaml should move to $home/.vizlab/profile.yaml
And add instructions to readme to indicate this needs to be done once for new users, and in vignette.

**Comment by lindsaycarr** [comment]
closed with #62
